java8/src/main/java/com/dslplatform/json/processor/CompiledJsonAnnotationProcessor.java

```diff
@@ -393,6 +393,8 @@ private static void buildCode(
 			final String generatePackage = generateFullClassName.substring(0, dotIndex);
 			code.append("package ").append(generatePackage).append(";\n\n");
 		}
+		code.append("import javax.annotation.Generated;\n\n");

```

Comment:
Should this use javax.annotation.processing.Generated for Java 9+ ?

Also its better to fully reference annotation instead of doing import